addons/l10n_ar/models/account_move.py

```diff
@@ -24,7 +24,7 @@ def _l10n_ar_get_document_number_parts(self, document_number, document_type_code
         ' identify the type of responsibilities that a person or a legal entity could have and that impacts in the'
         ' type of operations and requirements they need.')
 
-    l10n_ar_currency_rate = fields.Float(copy=False, digits=(16, 6), readonly=True, string="Currency Rate")
```

Comment:
@jjscarafia Ok for the rest, but I don't see why to remove the digits= stuff as it has enough precision and it might have consequences in stable (without it defaults to 2 digits normally)

This field was a mistake from the start.  I am checking if we will get a field that we can also set in master, so we can remove this one more easily as it is used in quite some code.  (but this field needs to go for master whatever)

Reply:
https://github.com/odoo/odoo/pull/133299